src/pegin/components/create/PegInForm.vue

```diff
@@ -144,16 +144,11 @@ export default defineComponent({
     const selectedAccountBalance = useGetter<SatoshiBig>('pegInTx', constants.PEGIN_TX_GET_SELECTED_BALANCE);
 
     const enoughAmountFlyover = computed(() => {
-      const quote = selectedQuote.value?.quote;
-      if (!quote) {
+      if (!selectedQuote.value) {
```

Comment:
create a function like isValid internally on `selecterQuote` object, verifying if value is not empty or 0 or null.